docs/UserGuide.md

```diff
@@ -153,6 +153,18 @@ Examples:
 * `list` followed by `delete 2` deletes the 2nd person in the contacts list.
 * `find Betsy` followed by `delete 1` deletes the 1st person in the results of the `find` command.
 
+###Deleting multiple contacts simultaneously : `massdelete`
+
+Deletes all contacts within the specified index range.
+
+Format: `massdelete start/INDEX end/INDEX`
```

Comment:
Perhaps the command format could be simplified to `massdelete s/INDEX e/INDEX` instead of having to type out the full words for start and end?

Reply:
"e" is already used for email. I'm not sure if I should reuse the same letter for a different function.